index.Rmd

```diff
@@ -1,105 +1,594 @@
+## Results - NYS Average Rates Within <i> I. scapularis</i> Populations
 
-Show tables, plots, etc. and describe them.
+```{r plot1view, echo=F,message=F}
+plot1
+```
+
+Pathogen prevalence in NYS shows a general trend of increase over time. Average rates of each pathogen are consistent with expectations where <i> B. burgdorferi </i> shows the highest prevalence within <i> I. scapularis </i> populations in NYS. <i> A. phagocytophilum </i>, <i> B. microti </i>, and <i> B. miyamotoi </i> exhibit the second, third, and fourth highest prevalence, respectively. Additionally, prevalence of each pathogen is about half as much in nymphal ticks compared to adult ticks. 
+
+
+<br>
+
+## Results - County Average Rates Within <i> I. scapularis</i> Populations
 
-```{r, fig.width=6, fig.height=3, fig.cap="Map of completely random data"}
-m <- leaflet(data) %>% 
-  addTiles() %>% 
-  addCircleMarkers(~x, ~y, radius = ~size,color = ~as.factor(category)) %>% 
-  addPopups(~x[2], ~y[2], "Random popup")
-m  # a map with the default OSM tile layer
+```{r leafletview, echo=F,message=F}
+plot2
 ```
 
+General takeaways from the interactive county level map are the following:
 
-```{r}
-data %>% 
-  ggplot(aes(x=x,y=y,col=category))+
-  geom_point()
+1. Rates in the capital district and metropolitan areas of NYS are generally higher than the NYS average, and also have more complete data.
+2. The north county (Adirondack region) has generally prevalence numbers (likely due to low numbers of tick populations).
+3. In counties with complete data, pathogen prevalence generally has risen over time.
+
+<br>
+
+## Results - Forest Cover by County in New York State
+```{r fc_map_setup,results='hide',message=FALSE, warning=F,echo=F}
+choro=left_join(County_Boundaries,df1)
+choro$Percent_Forest_Cover=choro$Percent_Forest_Cover*100
 ```
 
+```{r fc_map,echo=F,message=F}
+tm_shape(choro)+
+  tm_graticules(lines=F)+
+  tm_polygons(col="Percent_Forest_Cover",
+              palette = "viridis", n = 5,
+              title="Forest Cover (%)",
+              border.col='black')+
+  tm_compass(position = c('left','bottom'))+
+  tm_scale_bar(position=c('left','bottom'))+
+  tm_layout(main.title = "Area Covered by Mixed/Deciduous Forest",main.title.position = 'center')
+```
+
+<br>
+
+## Results - Statistical Analysis
+```{r join_NLCDw_County,results='hide',message=FALSE, echo=F}
+adulttestingdata=left_join(Adultdata,df1)
+nymphtestingdata=left_join(Nymphdata,df1)
+```
+
+```{r examine_Forest_Cover,message=FALSE, echo=F}
+hist(df1$Percent_Forest_Cover,
+     xlab="Percent of County Area (Mixed/Deciduous Forest)",
+     main="Histogram of County Areas")
+```
+
+The percent area of each county covered by deciduous or mixed forest appears normally distributed.
+
+```{r adult_prevalence_viz,message=FALSE, echo=F}
+par(mfrow=c(2,2))
```

Comment:
these are fine, but also know that you can do this sort of thing with ggplots using `ggarrage()` in library(ggpubr)`

Reply:
or create with facets.